Automattic/wp-calypso
# G Suite: Update Pending ToS notices and dialog for Google Workspace

This pull request updates the dialog displayed when a user has not accepted the terms of service of a G Suite user yet in order to also support Google Workspace. It also refines the notices and the copy of the dialog with simpler and more accurate language:

Before | After
------ | -----
![B1](https://user-images.githubusercontent.com/594356/107267547-54a12000-6a47-11eb-8a98-ec3f3d889d0e.png) | ![A1](https://user-images.githubusercontent.com/594356/107267558-5965d400-6a47-11eb-8541-1bf06c64b059.png)


Before | After
------ | -----
![B2](https://user-images.githubusercontent.com/594356/107267568-5d91f180-6a47-11eb-96bc-95878fa7ecff.png)| ![A2](https://user-images.githubusercontent.com/594356/107267577-61be0f00-6a47-11eb-9afe-c2df89496f25.png)


Before | After
------ | -----
![B3](https://user-images.githubusercontent.com/594356/107267590-6682c300-6a47-11eb-8f9d-6f4a4ed148a4.png) | ![A3](https://user-images.githubusercontent.com/594356/107267603-6b477700-6a47-11eb-9d9a-3925f4d686cc.png)

Finally, this pull request fixes an error message, a file permission as well as a placeholder.

#### Testing instructions

1. Run `git checkout update/pending-tos-notice-dialog-for-google-workspace` and start your server, or open a [live branch](https://calypso.live/?branch=update/pending-tos-notice-dialog-for-google-workspace)
2. Log into a WordPress.com account with a Google Workspace account with inactivated users
3. Open the [`Email` page](http://calypso.localhost:3000/email)
4. Assert that you see a notice inviting you to activate the Google Workspace users
5. Click the `Finish Setup` button
6. Assert that G Suite is no longer mentioned in the dialog

> Note you will have to apply D56184-code to be able to reset a password of a Google Workspace user.
